lib/galaxy/managers/workflows.py

```diff
@@ -687,13 +687,23 @@ def update_workflow_from_raw_description(
             stored_workflow.latest_workflow = workflow
             workflow.stored_workflow = stored_workflow
 
+        data = raw_workflow_description.as_dict
```

Comment:
@ahmedhamidawan do you remember why you made that change ?

Reply:
The purpose of doing this was to be able to create a workflow with tags as part of the payload (`raw_workflow_description`); instead of first creating a workflow, and then updating it with tags. Therefore, here, I created this `data` dict to check for tags (similarly to how we do the same for name and annotation in the same function later in L701

Reply:
Oh wait, this was back when we used the `save_workflow_as` route to create workflows, so this was needed back then, Now we use the fast api POST workflows route, so this change might not be needed anymore as much,,,